app/views/hq/orphans/_form.html.erb

```diff
@@ -0,0 +1,151 @@
+        <%= f.label :mother_alive %>
+        <%= f.check_box :mother_alive %>
+      </div>
+      <div class="form-group">
+        <%= f.label :father_is_martyr %>
+        <%= f.check_box :father_is_martyr %>
+      </div>
+    </div>
+  </div>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">Original Address</h3>
+    </div>
+    <%= f.fields_for :original_address do |builder| %>
+			<div class="panel-body">
+				<div class="form-group">
+					<%= builder.label :city %>
+					<%= builder.text_field :city, class: "form-control" %>
+				</div>
+				<div class="form-group">
+					<%= f.label :province %>
+					<%= f.collection_select :original_address,
+					 @provinces, :id, :name, {},  {class: "form-control", disabled: !f.object.new_record?} %>
+				</div>
+        <div class="form-group">
+			    <%= builder.label :neighborhood %>
+			    <%= builder.text_field :neighborhood, class: "form-control" %>
+			  </div>
+        <div class="form-group">
+			    <%= builder.label :street %>
+			    <%= builder.text_field :street, class: "form-control" %>
+			  </div>
+			</div>
+    <% end %>
+  </div>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">Current Address</h3>
+    </div>
+    <%= f.fields_for :current_address do |builder| %>
+			<div class="panel-body">
+			  <div class="form-group">
+			    <%= builder.label :city %>
+			    <%= builder.text_field :city, class: "form-control" %>
+			  </div>
+			  <div class="form-group">
+			    <%= f.label :province %>
+			    <%= f.collection_select :current_address,
+			     @provinces, :id, :name, {},  {class: "form-control", disabled: !f.object.new_record?} %>
+			  </div>
+        <div class="form-group">
+			    <%= builder.label :neighborhood %>
+			    <%= builder.text_field :neighborhood, class: "form-control" %>
+			  </div>
+        <div class="form-group">
+			    <%= builder.label :street %>
+			    <%= builder.text_field :street, class: "form-control" %>
+			  </div>
+			</div>
+    <% end %>
+  </div>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">Additional Details</h3>
+    </div>
+    <div class="panel-body">
+      <div class="form-group">
+        <%= f.label :sponsored_by_another_org %>
+        <%= f.check_box :sponsored_by_another_org %>
+      </div>
+      <div class="form-group">
+        <%= f.label :minor_siblings_count %>
+        <%= f.text_field :minor_siblings_count, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :sponsored_minor_siblings_count %>
+        <%= f.text_field :sponsored_minor_siblings_count, class: "form-control" %>
+      </div>
+    </div>
+  </div>
+  <div class="form-group">
+    <%= f.submit class: 'btn btn-primary' %>
+    <%= link_to('Cancel',
+                (@orphan.id ? hq_orphan_path(@orphan.id) : hq_orphans_path),
```

Comment:
The form will only be used for updating existing orphan records, so it's safe to have 'Cancel' always take the user to the orphan show view. Also, please don't hard-code `.id` in path helpers - `link_to 'Cancel', hq_orphan_path(@orphan)`.